testbed/testbed/results.go

```diff
@@ -118,16 +118,16 @@ type CorrectnessResults struct {
 
 // CorrectnessTestResult reports the results of a single correctness test.
 type CorrectnessTestResult struct {
-	testName              string
-	result                string
-	duration              time.Duration
-	sentSpanCount         uint64
-	receivedSpanCount     uint64
-	assertionFailureCount uint64
-	assertionFailures     []*AssertionFailure
+	testName                     string
+	result                       string
+	duration                     time.Duration
+	sentSpanCount                uint64
+	receivedSpanCount            uint64
+	tracingAssertionFailureCount uint64
+	tracingAssertionFailures     []*TracingAssertionFailure
```

Comment:
I feel that for metrics we will not call that "Metricing" so maybe TraceAssertion and MetricAssertion to be consistent? or use the plural, up to you

Reply:
Agree, fixed (singular :)